app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs

```diff
@@ -1,5 +1,55 @@
-<div class="ui ordered list">
-  {{#each record.eventRoles as |eventRole|}}
-    <div class="item">{{eventRole.name}} ({{eventRole.event.name}})</div>
+{{#if record.isUserOrganizer}}
+  {{#each record.events as |event|}}
```

Comment:
This is not the right way to do this. You're basically looping through all events that user is a part of. Whether he is an organizer or not. Then you're looping through all the organizers of that event. And then you're matching the email address. So many requests and loops.